Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -92,14 +92,22 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
     .. group-tab:: minikube
 
-       Install minikube >= v1.5.2 as per minikube documentation: 
+       Install minikube >= v1.5.2 as per minikube documentation:
        `Install Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
 
        .. code-block:: shell-session
 
-          minikube start --network-plugin=cni
+          minikube start --network-plugin=cni --memory=4096
           minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
```

Comment:
I don't think this step is required. Cilium init script mounts the BPF fs if not already mounted. 
I just ran these commands - 

```
$ minikube start --network-plugin=cni
$ cilium install
$ minikube ssh -- mount | grep bpf
none on /sys/fs/bpf type bpf (rw,relatime)


```

Reply:
It should not, this is done via `cilium install` now: https://github.com/cilium/cilium-cli/blob/ce6aa0252795e5ec4909f39245bced0cc4cbdf6f/install/install.go#L685

Reply:
That's already mentioned in the commit description, no? ;)

Reply:
I've tested it locally. It works now, I'm not sure why it failed for me before :thinking: